### PR TITLE
can now disable the row highlight with a prop

### PR DESCRIPTION
**Pull request by gillwilliam**
- I think my editor removed some of the trailing whitespace
- any suggestions for the extra long line?

### CODE REVIEW

File: src/TableBody.js [hunk 1]
```diff
@@ -147,6 +147,7 @@ class TableBody extends React.Component {
           'hideBorderBottom',
           'rowHeight',
           'shouldRowUpdate',
+          'shouldHighlightRow'
```
Comment (mroyce) [inline src/TableBody.js]: need trailing comma here

File: yarn.lock [hunk 2]
```diff
@@ -5406,7 +5406,7 @@ rc@^1.1.7:
     minimist "^1.2.0"
     strip-json-comments "~2.0.1"
 
-react-dom@^16.3.0:
+react-dom@^16.8.0:
```
Comment (mroyce) [inline yarn.lock]: lol whoops

Comment (gillwilliam) [reply yarn.lock]: should I remove these changes?

Comment (mroyce) [reply yarn.lock]: nah these are good, i just didn't update it when i updated package requirements

**Pull request merged**